### DISCUSSION TITLE
Shop Receipt VAT Cost Always 0

### DISCUSSION THREAD

**Discussion by Esatyilmaz0** [etsy/open-api#816] [Q&A]
Hello, I am accessing the information of my receipts using the getShopReceipt Endpoint, but I cannot get the VAT cost I am trying to get for the European Union countries, whatever the result is, it returns 0. What can I do about it?

**Comment by cyhsujohn** [comment]
I also have the same issue. We're trying to identify the VAT etsy collected but we couldn't get a clear amount from the API response.
For UK orders, Etsy indicates VAT was collected but the total_vat_cost is still 0 and total_tax_cost seems to be the VAT.
